Casamari/MNC008.xml

```diff
@@ -135,85 +276,220 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    unit="chars" quantity="4"/> ተማኀፀነ፡</seg>
+                           </q>
+                        </item>
+
                         <item xml:id="a2">
-                           <desc type="PoemArke"/>
+                           <desc type="GuestText"/>
+                           <title ref="LIT2835RepCh119">Excerpt from the <foreign xml:lang="gez"
+                                 >Malkǝʾa Mikāʾel</foreign></title>
+                           <desc>The excerpt comprises one complete stanza of the hymn.</desc>
                            <locus from="2r"/>
-                           <desc>Rhymed ʿarke-poem (rhyme in -ኖ).
-                           </desc>
+                           <q xml:lang="gez">
+                              <seg>ሰላም፡ ለመታክፋቲከ፡ ዘረሰየ፡ ክዳኖ። ኤጲሞሰ፡ ዋካ፡ አራዘ፡ ተክህኖ። ሚከኤል፡ ፍንው፡ ለረድኤት፡
+                                 ወለአድኅኖ። ከመ፡ ሐረሳዊ፡ ዘይቀኒ፡ ተያፍኖ።መታክፈ፡ ፀርየ፡ ቅቅኒ፡ ወቅፃዕ፡ ዘባኖ።</seg>
+                           </q>
                         </item>
-                        </list>
+
+                        <item xml:id="a3">
+                           <desc type="Unclear">List of individuals</desc>
+                           <locus from="1v" to="2r"/>
+                           <desc>The list is crudely written in pencil and includes the following
+                              names: <roleName type="title">ʾAbbā</roleName> Walda ʾIyasus,
+                                 <roleName type="title">ʾAbbā</roleName> Gabra Wald, <roleName
+                                 type="title">Qes</roleName> Gabra Wald, Tasfā Bǝyy, <roleName
+                                 type="title">Qes</roleName> Dastā, <roleName type="title"
+                                 >Qes</roleName> Warqǝnǝh, <roleName type="title">Qes</roleName>
+                              Walda Yoḥannǝs, <roleName type="title">Qes</roleName> Śāhle Kidāna
+                              Māryām, Gabra Yoḥannǝs, Gabra Wald. The list begins and ends with
+                              unclear scribbles. The name of Bayyanačč Zāwde is written in ink on
+                                 <locus target="#2v"/>.</desc>
+                        </item>
+
+                        <item xml:id="e1">
+                           <desc>Corrections were made carelessly. Superfluous words (e.g. <locus
+                                 target="#4r #5v #10v"/>), or a text passage spanning several lines
+                              (e.g. <locus target="#8v"/>) were roughly crossed out. Missing
+                              characters and words were interlineally inserted by the main hand
+                              (e.g. <locus target="#4r #22r #47r"/>) or by secondary hands (e.g.
+                                 <locus target="#10v #23r #46v"/>), several lines were written over
+                              erased? text on <locus from="13r" to="13v"/>. </desc>
```

Comment:
I am not sure if there was a first text layer which has been erased and overritten. Only in the first lines of f. 13v it seems to be the case. The question mark shall express this uncertainty. What do you think?
![MNC-008_014](https://user-images.githubusercontent.com/28757489/102876942-3e62e480-4446-11eb-956c-a893dae3958d.jpg)
![MNC-008_013](https://user-images.githubusercontent.com/28757489/102876954-41f66b80-4446-11eb-8505-ed8150132043.jpg)

Reply:
Instead of `over erased? text` would write `possibly over erasure`, in brackets or not

Reply:
I see it now, but only on f. 13v in correspondence of the punctuation mark.


> Instead of `over erased? text` would write `possibly over erasure`, in brackets or not

It makes sense.